### PR TITLE
S2U-7 T&Q: Several fixes

### PR DESCRIPTION
**Pull request by stetsche**
- Renamed "N" column to "Total attempts"
- Fixed Total attempts vs Total attempts(n) display in export
- Fixed missing not displaying of shorter rows (and possible misalignment) in PDF export

https://sakaiproject.atlassian.net/browse/S2U-7

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 1]
```diff
@@ -916,7 +916,7 @@ else if (!qbean.getQuestionType().equals(TypeIfc.MATCHING.toString())) // matchi
       getImageMapQuestionScores(publishedItemTextHash, publishedAnswerHash, (List) scores, qbean, (List) text);
     }
 
-    long attemptCount = Optional.ofNullable(qbean.getN()).map(Long::valueOf).orElse(0L);
+    long attemptCount = Optional.ofNullable(qbean.getNumResponses()).map(Long::valueOf).orElse(0L);
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:919]: @mpellicer Don't laugh!

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:919]: Nahh, laughing is optional.

Optional.ofNullable(null).orElse("hahaha I told you")

**Review by mpellicer — APPROVED** [review]
Looks sensible to me, does this fix the latest comments from Mateu ?

**Comment by stetsche** [comment]
Yes, should be all resolved now!

**Comment by mpellicer** [comment]
> Yes, should be all resolved now!

Terrific work!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java (hunk 1 above)
Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:919]: if (true) {
return "You are just pretending its optional!";
}

**Pull request merged**